CHANGELOG.md

```diff
@@ -1 +1,160 @@
 # Changelog
+
+For official release notes for Docker Engine CE, visit the
+[release notes page](https://docs.docker.com/engine/release-notes/).
+
+## 20.10.0
+
+### Deprecation / Removal
+
+* Sterner warnings and deprecation notice for unauthenticated tcp access [moby/moby#41285](https://github.com/moby/moby/pull/41285)
+* Deprecate KernelMemory (`docker run --kernel-memory`) [moby/moby#41254](https://github.com/moby/moby/pull/41254) [docker/cli#2652](https://github.com/docker/cli/pull/2652)
+* Deprecate `aufs` storage driver [docker/cli#1484](https://github.com/docker/cli/pull/1484)
+* Deprecate host-discovery and overlay networks with external k/v stores [moby/moby#40614](https://github.com/moby/moby/pull/40614) [moby/moby#40510](https://github.com/moby/moby/pull/40510)
+* Deprecate Dockerfile legacy 'ENV name value' syntax, use `ENV name=value` instead [docker/cli#2743](https://github.com/docker/cli/pull/2743)
+* Remove deprecated "filter" parameter for API v1.41 and up [moby/moby#40491](https://github.com/moby/moby/pull/40491)
+* Disable distribution manifest v2 schema 1 on push [moby/moby#41295](https://github.com/moby/moby/pull/41295)
+* Remove hack MalformedHostHeaderOverride breaking old docker clients (<= 1.12) in which case, set `DOCKER_API_VERSION` [moby/moby#39076](https://github.com/moby/moby/pull/39076)
+* Remove "docker engine" subcommands [docker/cli#2207](https://github.com/docker/cli/pull/2207)
+* Remove experimental "deploy" from "dab" files [docker/cli#2216](https://github.com/docker/cli/pull/2216)
+* Remove deprecated `docker search --automated` and `--stars` flags [docker/cli#2338](https://github.com/docker/cli/pull/2338)
+* No longer allow reserved namespaces in engine labels [docker/cli#2326](https://github.com/docker/cli/pull/2326)
```

Comment:
I think this list is missing the removal of `--stream`, possibly.

Reply:
Yes we should add (let me also open a PR for the `deprecated.md`. Related pull requests are https://github.com/docker/cli/pull/2105 and https://github.com/moby/moby/pull/39983

Reply:
opened https://github.com/docker/cli/pull/2809